docs/src/guides/ibexa/deploy.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Get started
+weight: -10
+description: Learn how to use Ibexa DXP on Platform.sh.
+---
+
+Ibexa DXP is a Composer-based PHP CMS, and as such fits well with the Platform.sh model.
+As a Symfony-based application its setup is very similar to Symfony.
+
+Ibexa DXP has come pre-configured for use with Platform.sh since its predecessor, eZ Platform 1.13.
+Version 2.5 and later is recommended.
+Those are the only versions that are supported.
+Appropriate Platform.sh configuration files are included in the Ibexa DXP application itself, but may be modified to suit your particular site, if needed.
+
+## Cache and sessions
+
+By default, Ibexa DXP is configured to use a single Redis instance for both the application cache and session storage.
+You may optionally choose to use a separate Redis instance for session storage in case you have a lot of authenticated traffic (and thus there would be many session records).
+
+To do so, uncomment the `redissession` entry in the `.platform/services.yaml` file
+and the corresponding relationship in the `.platform.app.yaml` file.
+The bridge code that is provided with eZ Platform 1.13 and later automatically detects the additional Redis service and use it for session storage.
+
+On a Dedicated instance, we strongly recommend using two separate Redis instances for Cache and Sessions.
+The service and relationship names that ship with the default Platform.sh configuration in Ibexa DXP should be used as-is.
+To ensure the development environment works like Production, uncomment the `redissession` entry in the `.platform/services.yaml` file and the corresponding relationship in the `.platform.app.yaml` file.
+The bridge code that's provided with eZ Platform 1.13 and later
+automatically detects the additional Redis service and uses it for session storage.
+
+By default, on Dedicated instances, both Cache and Session storage are in "persistent" mode,
+so that data isn't lost in case of a system or process restart.
+That reduces the potential for cache stampede issues or inadvertently logging people out.
+
```

Comment:
```suggestion
For more information, see [service types](/configuration/services/redis.md#service-types) 
```

Reply:
That section is explaining the difference between ephemeral and not. I'm not sure the relevance is immediately clear and I wouldn't want to get into restructuring a part of this to make it clear.